apps/settings/test/marionette/tests/root_settings_test.js

```diff
@@ -106,28 +106,6 @@ marionette('check root panel settings', function() {
           'geolocation.enabled should be false');
       });
     });
-
```

Comment:
Can we add a USB storage specific test that does similar checking but in the USB storage panel?